core/jazz_ui/protractor.conf.js

```diff
@@ -1,6 +1,6 @@
 // Protractor configuration file, see link for more information
 // https://github.com/angular/protractor/blob/master/lib/config.ts
-
+SELENIUM_PROMISE_MANAGER: false
```

Comment:
add comment what this configuration does

Reply:
this is not needed in test suit. I removed it from the suit.